pkg/vsphere/simulator/datastore.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package simulator
+
+import (
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/vmware/govmomi/vim25/methods"
+	"github.com/vmware/govmomi/vim25/mo"
+	"github.com/vmware/govmomi/vim25/soap"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+type Datastore struct {
+	mo.Datastore
+}
+
+func (ds *Datastore) RefreshDatastore(*types.RefreshDatastore) soap.HasFault {
+	r := &methods.RefreshDatastoreBody{}
+
+	info := ds.Info.GetDatastoreInfo()
+
+	buf, err := exec.Command("df", "-k", info.Url).Output()
```

Comment:
Yes, I have done that before, in gosigar, avoided it here for now since darwin is a different syscall: https://github.com/cloudfoundry/gosigar/blob/master/sigar_darwin.go#L166

I could also just remove the RefreshDatastore method for now, we don't need it for testing yet, just nice to see with govc datastore.info against vcsim.  Later we could consider using gosigar or something like to fill in some of the other data, such as HostSystem cpu, mem, etc.